pages/1.11/installing/production/advanced/index.md

```diff
@@ -6,8 +6,20 @@ menuWeight: 10
 excerpt: Using the Advanced Installer to create DC/OS clusters
 ---
 
-With this installation method, you package the DC/OS distribution yourself and connect to every node manually to run the DC/OS installation commands. This installation method is recommended if you want to integrate with an existing system or if you don’t have SSH access to your cluster.
+A DC/OS installation process requires a Bootstrap node, Master node, Public Agent node, and a Private Agent node. You can view [nodes][9] documenation for more information.
+Using advanced installation method, you can package the DC/OS distribution yourself and connect to every node manually to run the DC/OS installation commands. This installation method is recommended if you want to integrate with an existing system or if you don’t have SSH access to your cluster.
 
+# Advanced Installation Workflow
+
+Use the following steps to install DC/OS:
```

Comment:
This list should be paralleled in the structure of the doc (reflected in the right-hand nav bar) or we should remove it.